pom.xml

```diff
@@ -221,6 +228,18 @@
             <scope>test</scope>
             <version>1.4.194</version>
         </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
+            <version>1.12.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>kafka</artifactId>
+            <version>1.11.3</version>
```

Comment:
Version [`1.12.0`](https://repo1.maven.org/maven2/org/testcontainers/kafka/1.12.0/) has been released so we can keep the `org.testcontainers`'s artifacts version aligned to `1.12.0`

Reply:
FYI there is also `org.testcontainers:testcontainers-bom`